src/core/models/filter.js

```diff
@@ -192,6 +192,16 @@ export default class Filter {
     return Filter._fromMatcher('builtin.location', '$near', { lat, lng, radius });
   }
 
+  /**
+   * Create a new RADIUS_FILTER.
+   * @param {number} radius
+   */
+  static locationRadius (radius) {
```

Comment:
This method doesn't do much besides call the Filter constructor. Is it really necessary? I'm hesitant to clutter the Filter class with more of these static method. There are already a lot.

Reply:
removed